logicrunner/executionregistry/execution_registry.go

```diff
@@ -107,23 +107,25 @@ func (r *executionRegistry) Done(transcript *common.Transcript) bool {
 }
 
 // constructs all StillExecuting messages
-func (r *executionRegistry) OnPulse(_ context.Context) []insolar.Message {
+func (r *executionRegistry) OnPulse(_ context.Context) []payload.Payload {
 	r.registryLock.Lock()
 	defer r.registryLock.Unlock()
 
 	// TODO: this should return delegation token to continue execution of the pending
-	messages := make([]insolar.Message, 0)
+	messages := make([]payload.Payload, 0)
 	if len(r.registry) != 0 {
 		requestRefs := make([]insolar.Reference, 0, len(r.registry))
 		for requestRef := range r.registry {
 			requestRefs = append(requestRefs, requestRef)
 		}
 
-		messages = append(messages, &message.StillExecuting{
-			Reference:   r.objectRef,
-			Executor:    r.jetCoordinator.Me(),
-			RequestRefs: requestRefs,
-		})
+		if len(requestRefs) > 0 {
```

Comment:
this is bogus check. it can not be empty.

Reply:
it can, at least in tests

Reply:
it's impossible. you have `if len(registry) != 0` a few lines above.